core/arch/arm/arm.mk

```diff
@@ -5,6 +5,10 @@ CFG_CORE_TZSRAM_EMUL_SIZE ?= 458752
 CFG_LPAE_ADDR_SPACE_SIZE ?= (1ull << 32)
 
 CFG_MMAP_REGIONS ?= 13
+ifeq ($(CFG_CORE_ASLR),y)
+# 1 extra region for the identity map
+CFG_MMAP_REGIONS := $(CFG_MMAP_REGIONS) + 1
```

Comment:
This syntax is problematic because I think the two following commands will behave differently: `CFG_MMAP_REGIONS=13 make` and `make CFG_MMAP_REGIONS=13` (and until now we have avoided that).
Also it is a bit weird that the end value can be different than the one explicitly provided.

How about something like this instead?
```make
ifeq ($(CFG_CORE_ASLR),y)
IDMAP_REGIONS := 1
else
IDMAP_REGIONS := 0
endif
CFG_MMAP_REGIONS ?= 13 + $(IDMAP_REGIONS)
```

Reply:
Well spotted. Perhaps this should be handled directly with an ifdef where `static_memory_map` is declared instead to keep things simple.